pensjon-brevbaker/src/main/kotlin/no/nav/pensjon/brev/maler/adhoc/GjenlevendeInfoEtter1970.kt

```diff
@@ -73,38 +73,38 @@ object GjenlevendeInfoEtter1970 : AutobrevTemplate<Unit> {
+                    English to "Do you need help getting a new job or more work?",
                 )
             }
 
             paragraph {
                 text(
                     Bokmal to "NAV tilbyr ulike tjenester og støtteordninger for deg som trenger hjelp til å få jobb. Du kan finne jobbsøkertips og lenke til ledige jobber på $ARBEID_URL.",
                     Nynorsk to "NAV tilbyr ulike tenester og støtteordningar for deg som treng hjelp til å få jobb. Du kan finne jobbsøkjartips og lenkje til ledige jobbar på $ARBEID_URL.",
-                    English to "The Norwegian Labour and Welfare Administration (NAV) offers various services and support schemes for those who need help to find employment. You can find jobseeker tips and links to available jobs at $ARBEID_URL.",
+                    English to "The Norwegian Labour and Welfare Administration (NAV) offers various services and support schemes for those who need help finding a job. You can find job search tips and a hyperlink to open jobs online at $ARBEID_URL.",
                 )
             }
             paragraph {
                 text(
                     Bokmal to "Vi kan snakke med deg om ulike muligheter i arbeidsmarkedet, utdanning eller andre tiltak.",
                     Nynorsk to "Vi kan snakke med deg om ulike moglegheiter i arbeidsmarknaden, utdanning eller andre tiltak.",
-                    English to "We can give advice about the opportunities available to you, whether it be employment, education or other initiatives.",
+                    English to "We are available for a conversation about the various opportunities available to you in the labour market, education, or other initiatives.",
                 )
             }
 
             paragraph {
                 text(
                     Bokmal to "Hvis du trenger utdanning som blir godkjent av NAV, eller du får arbeidstrening via NAV, kan du søke om skolepenger og tilleggsstønad. Du finner mer informasjon om disse stønadene på $GJENLEVENDE_TILLEGGSSTOENADER_URL.",
                     Nynorsk to "Viss du treng utdanning som blir godkjent av NAV, eller viss du får arbeidstrening via NAV, kan du søkje om skolepengar eller tilleggsstønad. Du finn meir informasjon om desse stønadene på $GJENLEVENDE_TILLEGGSSTOENADER_URL.",
-                    English to "If you need to attain additional education and NAV approves your education plan, or if you are participating in employment training from NAV, you can apply to have tuition fees or other supplementary benefits covered by NAV. You can find more information about these benefits at $GJENLEVENDE_TILLEGGSSTOENADER_URL.",
+                    English to "If you are undertaking an education that NAV means is necessary for you to get a job and support yourself, you may get additional benefits. The same applies if you are registered as a jobseeker with NAV. You can find more information about these benefits at $GJENLEVENDE_TILLEGGSSTOENADER_URL.",
```

Comment:
The english version contains more information then  the other languages. Should it be translated back? i guess we don't have time but if this weren't an ADHOC letter, it would be weird. Did you fact check with someone that this is correct?